### PR TITLE
[PBW-5528]Discovery Up next appears fix after starting video playback

### PR DESCRIPTION
**Pull request by shruthi17**
Also, tested on the devices on which this manifested earlier and are not reproducible further.

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -406,7 +406,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
-    showUpNextScreenWhenReady: function(currentPlayhead, duration) {
+    howshowUpNextScreenWhenReady: function(currentPlayhead, duration) {
```
Comment (mistermendez) [inline js/controller.js]: TYPO: this should be `showUpNextScreenWhenReady`

File: js/controller.js [hunk 2]
```diff
@@ -420,7 +420,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.state.upNextInfo.timeToShow = timeToShow;
 
-      if (duration - currentPlayhead <= timeToShow &&
+if (duration!==0 &&- currentPlayhead <= timeToShow &&
```
Comment (mistermendez) [inline js/controller.js]: Should this be?

```
if (duration !== 0 &&
  duration - currentPlayhead <= timeToShow &&
```

Comment (mistermendez) [inline js/controller.js]: Also fix the indentation

File: js/controller.js (hunk 1 above)
Comment (shruthi17) [inline js/controller.js]: will fix it, thanks:)

File: js/controller.js (hunk 2 above)
Comment (shruthi17) [inline js/controller.js]: yeah it has to be ,  if (duration !== 0 && duration - currentPlayhead <= timeToShow &&

will fix it.

File: js/controller.js [hunk 3]
```diff
@@ -417,10 +417,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         // time to show is based on percentage of duration from the beginning
         timeToShow = (1 - parseInt(stringTimeToShow)/100) * duration;
       }
-
       this.state.upNextInfo.timeToShow = timeToShow;
 
-      if (duration - currentPlayhead <= timeToShow &&
+       if (duration!==0 && duration- currentPlayhead <= timeToShow &&
```
Comment (mistermendez) [inline js/controller.js]: - remove extra space in front of `if`
- add spaces around operator, should be `duration !== 0`
- keep each condition on separate line

**Comment by mistermendez** [comment]
👍

**Pull request merged**